deprecated-dune-v1-abstractions/ethereum/aztec_v2/contract_labels.sql

```diff
@@ -28,6 +29,13 @@ insert into aztec_v2.contract_labels
 ('Curve',                'Bridge',           '1.0',     'CurveStEth Bridge',            '\x0031130c56162e00a7e9c01ee4147b11cbac8776'::bytea)
 ;
 
+insert into aztec_v2.contract_labels 
```

Comment:
fyi this table has been updated on v1 engine, the PR can focus on v2 entries now

Reply:
thanks!